### ISSUE TITLE
Feature Request: ipnetwork.Inverse

### ISSUE THREAD

**Issue by rgreasley** [lduchosal/ipnetwork#118]
Hello,

Would it be possible to implement an "ipnetwork.Inverse" to get the inverse netmask?
Let's say the netmask is 255.255.255.0, the inverse would be 0.0.0.255.

Thanks.

**Comment by ajstadlin** [comment]
```cs
    /// <summary>
    /// Netmask Inverse
    /// example: _netmask == 255.255.255.0 returns the inverse 0.0.0.255
    /// </summary>
    public IPAddress NetmaskInverse
    {
      get
      {
        return IPNetwork.ToIPAddress(~this._netmask, this._family);
      }
    }
```

**Comment by lduchosal** [comment]
Implemented as WildcardMask in 2.6.437+
https://en.wikipedia.org/wiki/Wildcard_mask
